src/Features/CSharp/Portable/ConvertSwitchStatementToExpression/ConvertSwitchStatementToExpressionDiagnosticAnalyzer.Analyzer.cs

```diff
@@ -211,6 +211,9 @@ private static SyntaxKind Aggregate<T>(SyntaxKind seed, SyntaxList<T> nodes, Fun
 
             public override SyntaxKind VisitAssignmentExpression(AssignmentExpressionSyntax node)
             {
+                if (node.Right is RefExpressionSyntax)
+                    return default;
```

Comment:
Add braces.

Reply:
IDE style now allows for this explicitly.